locales/de/common.json

```diff
@@ -1 +1,3 @@
-{}
+{
+  "ImprintAndContact": "Impressum und Kontakt"
```

Comment:
I think maintaining all these language files seperately is more work than just one file per language. To be considered: There are about 10 languages to support ~ if we have ten different sections like footer, header,... we got 100 files which is very messy. Or do we have some other tools to manage these files?